test/controllers/api/v1/stats/channels_controller_test.rb

```diff
@@ -0,0 +1,35 @@
+require "test_helper"
+require "shared/mailer_test_helper"
```

Comment:
deadcode

Reply:
Fixed, but `require "test_helper"` is needed .